drivers/at/at.c

```diff
@@ -22,39 +23,77 @@
 #define AT_PRINT_INCOMING (0)
 #endif
 
-static void _isrpipe_write_one_wrapper(void *_isrpipe, uint8_t data)
+#if defined(MODULE_AT_URC_ISR_LOW)
+#define AT_EVENT_PRIO EVENT_PRIO_LOW
+#elif defined(MODULE_AT_URC_ISR_MEDIUM)
+#define AT_EVENT_PRIO EVENT_PRIO_MEDIUM
+#elif defined(MODULE_AT_URC_ISR_HIGH)
+#define AT_EVENT_PRIO EVENT_PRIO_HIGH
+#endif
+
+#if defined(MODULE_AT_URC_ISR)
+void _event_process_urc(event_t* _event)
 {
-    isrpipe_write_one(_isrpipe, data);
+    at_dev_t *dev = (at_dev_t*)container_of(_event, at_dev_t, event);
+    at_process_urc(dev, 1000);
+}
+#endif
+
+static void _isrpipe_write_one_wrapper(void *_dev, uint8_t data)
+{
+    at_dev_t *dev = (at_dev_t*) _dev;
```

Comment:
```suggestion
    at_dev_t *dev = (at_dev_t *) _dev;
```
There might be others

Reply:
Fixed